stan/math/prim/scal/prob/bernoulli_cdf.hpp

```diff
@@ -19,7 +19,21 @@
 namespace stan {
   namespace math {
 
-    // Bernoulli CDF
+    /**
+     * CDF of the Bernoulli distribution. If vectors of matching lengths
+     * are supplied, returns the product of the probabilities.
+     *
+     * @param n (sequence of) integers, must be 0 or 1
+     * @param theta (sequence of) probability parameters
+     *
+     * @tparam T_n type of integers
```

Comment:
replying to all of this inline but yes, making these fixes and expanding the scope a bit.